app-text/al-anvar/additional-files/al-anvar.rdef.in

```diff
@@ -0,0 +1,42 @@
+
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = @MINOR@,
+
+	variety = B_APPV_FINAL,
+	internal = 0,
+	short_info = "Al-Anvar",
+	long_info = "@LONG_INFO@"
+};
+
+resource app_signature "@APP_SIGNATURE@";
+
+resource file_types message {
+	"types" = "application/x-nvr"
+};
+
+resource vector_icon {
+	$"6E6369660B04018003010000020006023CC800389C28B89C283CC80049DBD7C6"
+	$"AAE100BAECFFFF008FFF020006023CC800389C28B89C283CC80049DBD7C6AAE1"
+	$"0075C2FFFF02366D020016023C5D733B1C57BB1C573C5D734BAB5648A8587157"
+	$"26B502001602BC54813B3720BB3720BC548149B0984B053BFFBE00FF02000602"
+	$"3CC800389C28B89C283CC80049DBD7C6AAE1000000FFFFF2F2F202001602BC62"
+	$"2BBB97633B9D05BC65E847C6844C30460057FFF402001602BBBEF1BA971F3AB9"
+	$"A5BBF0DE482F564BDE8F0057FFF402001602BC591EBB29993B2999BC591E481A"
+	$"B84BD27A0057FFF405000B0A05445D495D603E5A3B5B3F0607A62B223C49445B"
+	$"5A3E5D375D3A5D345A3139250A04243B444B5A313A260A0433323542483D482D"
+	$"0606AE0B444B4652464E4656445B5A3E5C375C3B5C345A310605E602223C4944"
+	$"5B46534657464F444B0802425625470802B642C1E6C0BCC70708022540434E02"
+	$"21BF1EBEE9BF1EBEE9BF1EBEE9BF25BF26BF25BF26BF25BF7EBECFC020BF09BF"
+	$"D1BE99C06FBDF9C0CCBE4FC0A7BDA0C0F1BD24C0C5BD5AC0EFBCEBC09EBCD0C0"
+	$"01BCD0C05ABCD0BF62BD8DBE1EBD0FBEC1BE0ABD7EBF7CBCACBEAFBD02C039BC"
+	$"5EC14EBC85C0D5BC51C1C5BCB9C203BDB4C203BD1DC203BE1BC1B2BF14C1E8BE"
+	$"90C17ABF95C041C133C101C048BF83C21DBEE3C323BF0DC2C1BEB5C385BE9FC4"
+	$"25BE9FC3D9BE9FC462BED1C4AABEB2C48C3EC4C9BF49C4BBBF1AC4CFBF74C4A8"
+	$"BFD6C423BFA5C476BFD6C423C005C42CC005C42CBFF3C47EBFA5C504BFD6C4C3"
+	$"BF73C542BF03C587BF40C56EBEBCC5A4BE57C57FBE81C5A1BE2AC558BE16C4B7"
+	$"BE16C517BE16C45DBE5AC389BE2BC3F8BE88C318BF7CC1A2BEE7C274C00DC0CE"
+	};
```

Comment:
@mazbrili, the only remaining issue is the vector icon.

BTW, in case you lost the icon we can simply remove it from the `al-anvar.rdef.in` file and then merge your PR.

Don't worry if you can't fix the icon right now. There is no urgency at all. We can wait until you are able to update the PR or, if you prefer, until you copy/paste to a comment the bloc which defines the icon.

Reply:
@fbrosson i'm sure have been test the icon..maybe im wrong when copy-pasting..
 i'm still not fit today do i got chicken-pox disease.. i cannot go anywhere technically.. until i recovered

Reply:
I double checked with your icon and then with another icon. When I build using your icon the app gets a default icon that Haiku displays for apps which have no icon. When I build using another icon I get that icon. So I believe the system detects an error in the icon which makes it impossible to show it and presents the default icon instead.
Yeah, there has probably been a partial or wrong copy/paste.
BTW, I guess you created an icon which does not look like a generic icon. Or did you grab a copy of a generic icon?

Anyway, you don't need to worry. We can keep this PR open as long as you wish.
Of course, if you would prefer to have this PR merged without the icon, this is also OK.
(I think it is slightly better to wait until we have the right icon, but I'm OK with merging without the icon.)
So, here's what we can do:
1) Forget about this PR, take care of yourself, and we'll keep this PR open.
2) At any time, if you wish to have this PR merged, either with an icon, or without the icon, you just need to tell us and I'll do whatever you wish.

We know you need to recover so don't worry. There is no deadline :)

Reply:
@fbrosson i'have been update other icons.. please check.. sorry to make problem

Reply:
Excellent! Thanks @mazbrili!